deepfence_agent/tools/apache/deepfence/df-utils/cloud_metadata/cloud_metadata.go

```diff
@@ -329,6 +377,9 @@ func GetDigitalOceanMetadata(onlyValidate bool) (CloudMetadata, error) {
 	}
 	digitalOceanMetadata.PublicIP = publicIP
 	digitalOceanMetadata.PrivateIP = privateIP
+
+	digitalOceanMetadata.Tags = getUserDefinedTags()
```

Comment:
This is required here also https://github.com/deepfence/ThreatMapper/blob/7cc61a27b32e2360ce33bb0d452f4dfcb1d515f7/deepfence_agent/tools/apache/deepfence/df-utils/cloud_metadata/cloud_metadata.go#L425

Reply:
Done.